lib/storage/src/content_manager/consensus/persistent.rs

```diff
@@ -38,6 +38,8 @@ pub struct Persistent {
     /// Operations to applied, consensus consider them committed, but this peer didn't apply them yet
     #[serde(default)]
     pub apply_progress_queue: EntryApplyProgressQueue,
+    #[serde(default, skip_serializing_if = "Option::is_none")]
+    pub first_voter: Option<PeerId>,
```

Comment:
Why not call this `origin_peer_id`?

I feel like first voter doesn't strictly represent the very first peer ID we ever see.